File: 1018.py
n = int(input())

notas = [100, 50, 20, 10, 5, 2, 1]

print(n)
for i in notas:
    count = n // i
    print('{:.0f} nota(s) de R$ {},00'.format(count, i))
    n %= i

File: 1018.php
<?php
    $notas = array(100, 50, 20, 10, 5, 2, 1);

    $valor = readline();

    echo $valor . "\n";

    for ($i = 0; $i < 7; $i++) {
        $count =  $valor / $notas[$i];
        
        echo (int) $count . " nota(s) de R$ " . $notas[$i] . ",00\n";
        
        $valor %= $notas[$i];
    }
?>